### PR TITLE
Clean up VLMPipeline

### PR DESCRIPTION
**Pull request by Wovchena**
Address comments in https://github.com/openvinotoolkit/openvino.genai/pull/912

### CODE REVIEW

File: src/cpp/include/openvino/genai/vlm_pipeline.hpp [hunk 1]
```diff
@@ -65,37 +65,14 @@ class OPENVINO_GENAI_EXPORTS VLMPipeline {
...
-        ov::Core core=ov::Core{}
+        const ov::AnyMap device_config={}
     );
 
     /// @brief Default destructor.
     ~VLMPipeline();
 
     /// @brief Generate a response given a prompt and any number of
-    /// uint8 RGB images.
+    /// uint8 RGB images with [HWC] layout.
```
Comment (ilya-lavrenov) [inline src/cpp/include/openvino/genai/vlm_pipeline.hpp:75]: if I pass NHWC, what will happen?

Comment (Wovchena) [reply src/cpp/include/openvino/genai/vlm_pipeline.hpp:75]: 1HWC is actually expected. Should I enable both?

Comment (ilya-lavrenov) [reply src/cpp/include/openvino/genai/vlm_pipeline.hpp:75]: I suppose yes, maybe in PR with multiple images you can add support of NHWC ? where ov::Tensor represents multiple images and N > 1

Comment (Wovchena) [reply src/cpp/include/openvino/genai/vlm_pipeline.hpp]: Done in [7441a18](https://github.com/Wovchena/openvino.genai-public/pull/68/commits/7441a1860def9dfb8012ef976b0d8526ed4975d6). @popovaan, please, add tests for [NHWC] ([1HWC], [2HWC]) and [HWC] image layouts.

**Review by ilya-lavrenov — APPROVED** [review]

**Comment by ilya-lavrenov** [comment]
Can be closed

**Pull request closed without merging**